website/templates/institution.mako

```diff
@@ -0,0 +1,52 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Institution</%def>
+
+<%def name="container_class()">container-xxl</%def>
+
+<%def name="content()">
+% if disk_saving_mode:
+    <div class="alert alert-info"><strong>NOTICE: </strong>Forks, registrations, and uploads will be temporarily disabled while the OSF undergoes a hardware upgrade. These features will return shortly. Thank you for your patience.</div>
+% endif
+    <div id="inst">
+<div class="dashboard-header">
+    <div class="row" style="text-align: center">
+        <div class="col-sm-2"></div>
+        <div class="col-sm-3"><img class="img-circle" height="110" width="110" src=${ logo_path }></div>
+        <div class="col-sm-3">
+            <h2>${ name }</h2>
+            <h4><small class="hidden-sm">${description}</small></h4>
```

Comment:
Would it be better to check if there is content in the description first? So that this element doesn't take up space on the page if the institution doesn't have a description.